src/packagedcode/rubygems.py

```diff
@@ -255,7 +258,6 @@ def get_gem_metadata(location):
         if warnings:
             raise Exception('Failed to extract RubyGem .gem file.\n' + '\n'.join(warnings))
```

Comment:
I would rather not have newlines in this Exception.

Reply:
Done :)